pkgs/blast_repo/lib/src/tweaks/reformat_tweak.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:firehose/firehose.dart';
```

Comment:
What is blast_repo using from firehose? Is it something we can in-line w/o a lot of cost?

Reply:
The `Repository` class - something that is needed in both.